src/Console/Command/FixCommand.php

```diff
@@ -547,6 +400,27 @@ protected function execute(InputInterface $input, OutputInterface $output)
     }
 
     /**
+     * @param ConfigurationResolver $resolver
+     * @param InputInterface        $input
+     * @param OutputInterface       $output
+     *
+     * @return ReportInterface
+     */
+    protected function createReport(ConfigurationResolver $resolver, InputInterface $input, OutputInterface $output)
```

Comment:
If you don't need anything else - yes. No need for that huge dependency. it would be easier for testing as well.